src/components/site/SummaryList.js

```diff
@@ -1,20 +1,44 @@
 import React from "react";
+import PropTypes from "prop-types";
 
 const SummaryList = props => {
-  const { items = [] } = props;
+  const { id, items = [] } = props;
```

Comment:
in this case, i think you can completely remove id now. From this line and the prop-types. Will be completely optional, and probably won't have an id, but not sure.

Reply:
removed id